packages/components/src/systems/DraftPrescriptions/CoverageOptions/OtherCoverageOptionsList.tsx

```diff
@@ -0,0 +1,80 @@
+import { CoverageOption, PrescriptionFormData } from '../../PrescribeProvider';
+import { For } from 'solid-js';
+import { CoverageOptionSummary } from './CoverageOptionSummary';
+import Button from '../../../particles/Button';
+import Card from '../../../particles/Card';
+import { format } from 'date-fns';
+
+export type OtherCoverageOptionsListProps = {
+  coverageOptions: CoverageOption[];
+  handleSwapToOtherPrescription: (other: PrescriptionFormData) => void;
+};
+
+export function OtherCoverageOptionsList(props: OtherCoverageOptionsListProps) {
+  return (
+    <>
+      <div>Other Options</div>
+      <For each={props.coverageOptions}>
+        {(coverageOption) => (
+          <AlternativeCoverageOptionListItem
+            coverageOption={coverageOption}
+            handleSwapToOtherPrescription={props.handleSwapToOtherPrescription}
+          />
+        )}
+      </For>
+    </>
+  );
+}
+
+export type AlternativeCoverageOptionListItemProps = {
+  coverageOption: CoverageOption;
+  handleSwapToOtherPrescription: (alternative: PrescriptionFormData) => void;
+};
+
+export function AlternativeCoverageOptionListItem(props: AlternativeCoverageOptionListItemProps) {
```

Comment:
nit, this is called `AlternativeCoverage` but the one above is called `OtherCoverage`